src/pymor/bindings/fenics.py

```diff
@@ -471,7 +471,7 @@ def visualize(self, U, m, title='', legend=None, filename=None, block=True,
                 if suffix not in supported:
                     msg = ('FenicsVisualizer needs a filename with a suffix indicating a supported backend\n'
                            + f'defaulting to .pvd (possible choices: {supported})')
-                    self.logger.warn(msg)
+                    self.logger.warnings(msg)
```

Comment:
```suggestion
                    self.logger.warning(msg)
```

At least my logger objects don't have a `warnings` method ..

Reply:
Good thing our tests caught that :roll_eyes: